### DISCUSSION TITLE
Should we push the code to the main or any other branch? for the Gari network web3 project

### DISCUSSION THREAD

**Discussion by akashgreninja** [community/community#51052] [GitHub Education]
They have not mentioned which branch should we oush our code to

**Comment by Ntshangase** [comment]
Hey,

I suggest that you contact you project mentor as they may have specific ways to contribute to their project.

But Usually when working on project's with other, it's recommended that you create a branch and make your changes there then after your branch has been reviewed it will be merged into the main branch.

**Reply by Ntshangase** [reply]
@akashgreninja if you found this helpful, you can "Mark as Answer" to help other people facing similar situations
